src/content/docs/en/pages/devtools/azion-edge-runtime/api-reference/object-storage/object-storage.mdx

```diff
@@ -0,0 +1,379 @@
+---
+title: Object Storage API 
+description: The Edge Functions Storage API is an interface that allows access to the Azion Object Storage. Through this interface, it's possible to read and write data in the storage and its buckets.
+meta_tags: edge function, edge computing, storage, jamstack, apis
+namespace: documentation_runtime_storage
+permalink: /documentation/devtools/runtime/api-reference/storage/
+menu_namespace: runtimeMenu
+---
+
+The [Edge Functions](/en/documentation/products/edge-application/edge-functions/) storage API is an interface that allows access to the Azion Object Storage. Through this interface, it's possible to read and write data in the storage and its buckets.
+
+## Class instantiation
+
+
+To leverage this API, you need to import and instantiate the `Storage` class, passing the bucket's name.
+
+### Syntax
+
+Import: 
+
+```javascript
+import Storage from "azion:storage";
+```
+
+Instantiate: 
+
+```js
+const storage = new Storage(bucket);
+```
+
+### Parameters
+
+| Parameter | Type | Description |
+| - | - | - |
+| `bucket` | string | Name of a bucket created during an [Azion CLI](/en/documentation/products/azion-cli/overview/) deploy or through [RTM](/en/documentation/products/guides/getting-to-know-real-time-manager/). |
+
+### Return value
+
+Storage object used to access Azion Object Storage.
+
+---
+
+## Methods
+
+### async Storage.put
+
+The `Storage.put` method is used to insert a new object into the storage.
+
+:::note
+This is an async method, and must be used with `await` or `event.waitUntil`.
+:::
+
+#### Syntax
+
+```js
+async Storage.put(key, value, options)
+```
+
+#### Parameters
+
+| Parameter | Type | Description |
+| - | - | - |
+| `key` | string | Identifier that allows the search for an object in the storage. |
+| `value` | ArrayBuffer ou ReadableStream | Content of the object being stored. In case `value` implements a [ReadableStream](https://developer.mozilla.org/en-US/docs/Web/API/ReadableStream), the Stream APIs can be used and the option `content-lenght` is required. |
```

Comment:
```suggestion
| `value` | ArrayBuffer ou ReadableStream | Content of the object being stored. In case `value` implements a [ReadableStream](https://developer.mozilla.org/en-US/docs/Web/API/ReadableStream), the Stream APIs can be used and the option `content-lenght` is required |
```